src/Parser/Mage/Arcane/Modules/ManaChart/TabComponent/ManaLevelGraph.js

```diff
@@ -0,0 +1,227 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Chart from 'chart.js';
+import {Line} from 'react-chartjs-2';
+
+import fetchWcl from 'common/fetchWclApi';
+
+import ManaStyles from 'Interface/Others/ManaStyles.js';
+
+const formatDuration = (duration) => {
+  const seconds = Math.floor(duration % 60);
+  return `${Math.floor(duration / 60)}:${seconds < 10 ? `0${seconds}` : seconds}`;
+};
+
+class Mana extends React.PureComponent {
+  static propTypes = {
+    reportCode: PropTypes.string.isRequired,
+    actorId: PropTypes.number.isRequired,
+    start: PropTypes.number.isRequired,
+    end: PropTypes.number.isRequired,
+    manaUpdates: PropTypes.array.isRequired,
+  };
+
+  constructor() {
+    super();
+    this.state = {
+      bossHealth: null,
+    };
+  }
+
+  componentWillMount() {
+    this.load(this.props.reportCode, this.props.actorId, this.props.start, this.props.end);
+  }
+  componentWillReceiveProps(newProps) {
+    if (newProps.reportCode !== this.props.reportCode || newProps.actorId !== this.props.actorId || newProps.start !== this.props.start || newProps.end !== this.props.end) {
+      this.load(newProps.reportCode, newProps.actorId, newProps.start, newProps.end);
+    }
+  }
+
+  load(reportCode, actorId, start, end) {
+    return fetchWcl(`report/tables/resources/${reportCode}`, {
+      start,
+      end,
+      sourceclass: 'Boss',
+      hostility: 1,
+      abilityid: 1000,
+    })
+      .then(json => {
+        console.log('Received boss health', json);
```

Comment:
stray `console.log`?

Reply:
i didnt add that, so it must be in the core manalevelgraph that healers use too. I can potentially remove it though.